src/pkg/services/m365/m365_test.go

```diff
@@ -145,6 +145,30 @@ func (suite *M365IntegrationSuite) TestSites() {
 	}
 }
 
+func (suite *M365IntegrationSuite) TestTeams() {
```

Comment:
Similar to moving the code from m365.go to groups.go, can you please move these tests to `groups_test.go` and give them a new suite?